jeeves/src/main/java/jeeves/utils/XmlRequest.java

```diff
@@ -25,15 +25,7 @@
 
 import jeeves.exceptions.BadSoapResponseEx;
 import jeeves.exceptions.BadXmlResponseEx;
-import org.apache.commons.httpclient.Cookie;
-import org.apache.commons.httpclient.Credentials;
-import org.apache.commons.httpclient.Header;
-import org.apache.commons.httpclient.HostConfiguration;
-import org.apache.commons.httpclient.HttpClient;
-import org.apache.commons.httpclient.HttpMethodBase;
-import org.apache.commons.httpclient.HttpState;
-import org.apache.commons.httpclient.NameValuePair;
-import org.apache.commons.httpclient.UsernamePasswordCredentials;
+import org.apache.commons.httpclient.*;
```

Comment:
http://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports Seems like a common complaint... based on the number of classes, so the only only it so set it to a high number.